src/app/teams/team-details/team-details.component.html

```diff
@@ -15,7 +15,7 @@
     <md-tab label="Info">
         <div class="header-content">
             <div class="header-text"><p>{{ team?.name }}</p><p class="team-info">[{{ ['coed', 'female', 'male'][team?.type] }}] {{ team?.info }}</p></div>
-            <button md-mini-fab *ngIf="isManager" [routerLink]="['edit']"><md-icon>edit</md-icon></button>
+            <button md-mini-fab *ngIf="isManager" [routerLink]="['/teams/edit', team.id]"><md-icon>edit</md-icon></button>
```

Comment:
Instead of this let's change the path to `:id/edit` for the edit view (/teams/{team_id}/edit)